knotx-example/knotx-example-app/config/application.conf

```diff
@@ -9,63 +10,107 @@ modules = [
   "actionKnot=io.knotx.knot.action.ActionKnotVerticle"
   "serviceAdapter=io.knotx.adapter.service.http.HttpServiceAdapterVerticle"
   "actionAdapter=io.knotx.adapter.action.http.HttpActionAdapterVerticle"
+
+  //TBD
   "repoMock=io.knotx.mocks.MockRemoteRepositoryVerticle"
   "serviceMock=io.knotx.mocks.MockServiceVerticle"
   "gatewayKnot=io.knotx.gateway.GatewayKnotVerticle"
   "reqProcessorKnot=io.knotx.gateway.RequestProcessorKnotVerticle"
   "respProviderKnot=io.knotx.gateway.ResponseProviderKnotVerticle"
 ]
 
-repo {
-  domain = localhost
-  port = 3001
-}
+########### Globals ###########
+global {
+  serverPort = 8092
+
+  snippetTagName = script
```

Comment:
Maybe it is the time to introduce the custom tag name like "knotx:snippet" :) ?

Reply:
We definitely have to change the default to e.g. `knotx:snippet`. But not yet, let's not break backward compability.

Reply:
I agree with skejven. Making it default to something else, not only break the backward compatibility, but also introduces a lot of refactors of tests - that anyway need to be rebuilt, changed, reorganized...